mapr/README.md

```diff
@@ -12,34 +12,49 @@ Follow the instructions below to install and configure this check for an Agent r
 
 The MapR check is included in the [Datadog Agent][2] package but requires additional setup operations.
 
-1. Create a `dd-agent` user with a password on every node in the cluster with the same `UID`/`GID` so it is recognized by MapR. See [Managing users and groups][10] for additional details.
-2. Install the Agent on every node you want to monitor.
-3. Install the library *mapr-streams-library* with the following command: `/opt/datadog-agent/embedded/bin/pip install --global-option=build_ext --global-option="--library-dirs=/opt/mapr/lib" --global-option="--include-dirs=/opt/mapr/include/" mapr-streams-python`. If you use Python 3 with Agent 6, replace `pip` by `pip3`.
-4. Add `/opt/mapr/lib/` to your `/etc/ld.so.conf` (or a file in `/etc/ld.so.conf.d/`). This is required to help the *mapr-streams-library* used by the Agent to find the MapR shared libraries.
-5. Generate a [long-lived ticket][8] for the `dd-agent` user.
-6. Make sure the ticket is readable by the `dd-agent` user.
-7. Configure the integration (see below).
+#### Prerequisites
 
-**Note**: If you don't have "security" enabled in the cluster, you can continue without a ticket.
+* [MapR Monitoring][3] needs to be running correctly.
+* You need an available [MapR user][4] (same name, password, uid and gid) with the 'consume' permission on the `/var/mapr/mapr.monitoring/metricstreams` stream. This may be an already existing user or a newly created user. If you want the user user to be called `dd-agent`, you have to create it before installing the Agent.
```

Comment:
```suggestion
* You have an available [MapR user][4] (with name, password, UID, and GID) with the 'consume' permission on the `/var/mapr/mapr.monitoring/metricstreams` stream. This may be an already existing user or a newly created user. If you want the user user to be called `dd-agent`, create it before installing the Agent.
```